### PR TITLE
add docker auth

### PR DESCRIPTION
**Pull request by Sotatek-TinnNguyen**
## Proposed changes

- Add docker auth to increase limit

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -340,6 +361,9 @@ jobs:
   build:
     docker:
       - image: klaytn/build_base:1.2-ubuntu18.04-go.1.20.6-solc0.8.13
+        auth:
+          username: $DOCKER_LOGIN
+          password: $DOCKER_PASSWORD
```
Comment (yoomee1313) [inline .circleci/config.yml:366]: What about using `test-executor` here?

**Review by yoomee1313 — COMMENTED** [review]
Lgtm except minor

File: .circleci/config.yml (hunk 1 above)
Comment (Sotatek-TinnNguyen) [reply .circleci/config.yml]: changed as your comment

**Review by blukat29 — APPROVED** [review]

**Review by yoomee1313 — APPROVED** [review]

**Comment by yoomee1313** [comment]
@Sotatek-TinnNguyen The review seems done. Can you merge this PR?

**Pull request merged**